### ISSUE TITLE
Explain why we are Creating an R project in Before we Start

### ISSUE THREAD

**Issue by JasonJWilliamsNY** [datacarpentry/R-genomics#51]
Episode: Before we start
We start the workshop by creating an RProject but we never explain why. We should either discuss a bit or else not create the project.

**Comment by AnnaWilliford** [comment]
Yes, please!  We need to explain the advantages of making Rprojects (if there are any)
